### PR TITLE
Release v0.11.0

### PR DESCRIPTION
**Pull request by fredcarle**
Release branch as of e4c59a965b9b864dd3759fe75836a8b6896d9688

Release tasks completed:

- [x] ran go mod tidy
- [x] regenerated in-repo docs (CLI and manpages)
- [x] bump BSL license parameters
- [x] update changelog
- [x] Made sure that the Playground can build

Only need to review the last commit :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+* DB transactions context ([#2513](https://github.com/sourcenetwork/defradb/issues/2513))
+* Add NormalValue ([#2404](https://github.com/sourcenetwork/defradb/issues/2404))
+* Clean up client/request package ([#2443](https://github.com/sourcenetwork/defradb/issues/2443))
+* Rewrite convertImmutable ([#2445](https://github.com/sourcenetwork/defradb/issues/2445))
+* Unify Field Kind and Schema properties ([#2414](https://github.com/sourcenetwork/defradb/issues/2414))
+* Replace logging package with corelog ([#2406](https://github.com/sourcenetwork/defradb/issues/2406))
+
+### Testing
+
+* Add more ACP integration tests ([#2583](https://github.com/sourcenetwork/defradb/issues/2583))
```
Comment (islamaliev) [inline CHANGELOG.md:47]: thought: not sure if this worth mentioning as we just added ACP

**Review by islamaliev — APPROVED** [review]
LGTM

**Review by shahzadlone — CHANGES_REQUESTED** [review]
todo: Likely want to also run `make mock` to update the following mock files:
- client/mocks/collection.go
- client/mocks/db.go
- db/fetcher/mocks/fetcher.go

Specially the mocks for client collection functions that got merged

File: CHANGELOG.md (hunk 1 above)
Comment (shahzadlone) [reply CHANGELOG.md:47]: Should have been a `(i)` commit. +1 on the thought to remove this from changelog.

File: README.md
Comment (shahzadlone) [inline README.md:1]: todo: Update the TOC (remove the early access section, add the ACP and other missing section).

todo: Update the Disclaimer

File: CHANGELOG.md (hunk 1 above)
Comment (fredcarle) [reply CHANGELOG.md]: ✅

File: README.md
Comment (fredcarle) [reply README.md:1]: I agree that the TOC and mocks should be kept up to date but in the future lets try to have those taken care of in their respective relevant PRs.

Comment (shahzadlone) [reply README.md:1]: We should definitely improve our process to have those always be updated in their respective PRs. Can discuss that elsewhere IMO.

**Pull request merged**